assets/main.js

```diff
@@ -1,16 +1,120 @@
+      bottle: 'https://cdnjs.cloudflare.com/ajax/libs/bottlejs/1.6.1/bottle.min',
+      vue: 'https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.4/vue',
+      vuex: 'https://cdnjs.cloudflare.com/ajax/libs/vuex/3.0.1/vuex.min',
+      bookingWizardComponents: 'https://unpkg.com/@rebelcode/booking-wizard-components@0.1.4/dist/lib.min',
+      formWizard: 'https://unpkg.com/vue-form-wizard/dist/vue-form-wizard',
+      axios: 'https://cdn.jsdelivr.net/npm/axios@0.18.0/dist/axios.min',
+      humanizeDuration: 'https://cdnjs.cloudflare.com/ajax/libs/humanize-duration/3.14.0/humanize-duration.min',
+      uiFramework: 'https://unpkg.com/@rebelcode/ui-framework@0.1.1/dist/static/js/uiFramework',
+      datepicker: 'https://cdn.jsdelivr.net/npm/vuejs-datepicker@0.9.26/dist/build.min',
+      lodash: 'https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.4/lodash.min',
+      textFormatter: 'https://unpkg.com/sprintf-js@1.1.1/dist/sprintf.min',
+      moment: 'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min',
+      momentTimezone: 'https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.17/moment-timezone-with-data-2012-2022.min',
+      momentRange: 'https://cdnjs.cloudflare.com/ajax/libs/moment-range/4.0.1/moment-range',
+    })
+  })
+
+  var dependenciesList = [
+    'bookingWizard',
+    'formWizard',
+    'textFormatter',
+    'bookingWizardComponents',
+    'uiFramework',
+    'bottle',
+    'vue',
+    'vuex',
+    'axios',
+    'humanizeDuration',
+    'cjs!datepicker',
+    'moment',
+    'momentTimezone',
+    'lodash',
+    'cjs!momentRange',
+    'stdLib'
+  ]
+
+  require(dependenciesList, function () {
+    var dependencies = {}
+
+    for (var i = 0; i < dependenciesList.length; i++) {
+      dependencies[dependenciesList[i].replace('cjs!', '')] = arguments[i]
+    }
+
+    var di = new dependencies.bottle()
+    defineServices(di, dependencies)
+
+    var container = new dependencies.uiFramework.Container.Container(di)
+    var app = new dependencies.uiFramework.Core.App(container)
+    app.init()
+  })
+
+  function defineServices (di, dependencies) {
+    var serviceList = dependencies.bookingWizard.services(dependencies, document)
+
+    serviceList['state'] = function () {
+      /**
+       * @typedef {Object} AppState
+       *
+       * @since [*next-version*]
+       *
+       * @property {string} applicationSelector Application selector for UI framework (UI framework will mount application to each of them).
+       * @property {{bookings: string, services: string}} apiBaseUrls Map of resources names to their urls.
+       * @property {string} initialBookingTransition Name of booking transition for creating booking.
+       * @property {object} datetimeFormats Map of datetime keys to datetime formats for formatting datetimes in application.
+       */
+
+      /**
+       * @var {AppState} EDDBK_WIZARD_APP_STATE
+       */
+      return window.EDDBK_WIZARD_APP_STATE

```

Comment:
What do you think about having a function to pass this into, instead of reading from a global variable?

Reply:
Not so important right now. Leave for later.